### ISSUE TITLE
Make 0.38.0 release after #1571 lands

### ISSUE THREAD

**Issue by SimonBiggs** [pymedphys/pymedphys#1574]
We'll also wait until @cedricbelanger has had the opportunity to put their name and institution logo on the front page so that it will also appear on the PyPI list for this release (https://pypi.org/project/pymedphys/).

@pchlap given you haven't done a release yet, did you want to do this one with me? I think the best way to go about it is we jump on a video call and we'll do all the release work within your environment, and I'll guide as we go. While I'm guiding, if one of us could take notes to act as a first pass on "how to do a release" documentation, that'd be awesome.

@sjswerdloff and @Matthew-Jennings you guys can feel free to be involved, just let me know :slightly_smiling_face:. 

---

Importantly, we'll need to prominently display the previous bug, its potential prior ramifications, and that it is fixed in this version within the change notes and any announcements for this release. Also we need to provide a bucket load of thanks to @cedricbelanger within the release notes and announcements as well as welcome him has a PyMedPhys contributor :slightly_smiling_face: :tada:

**Comment by pchlap** [comment]
> @pchlap given you haven't done a release yet, did you want to do this one with me? I think the best way to go about it is we jump on a video call and we'll do all the release work within your environment, and I'll guide as we go. While I'm guiding, if one of us could take notes to act as a first pass on "how to do a release" documentation, that'd be awesome.

Hi @SimonBiggs, that sounds great! I'd be keen to have a video call with you to go through this process. Perhaps one morning at 8am (I think the early start suits you better too?)?

**Comment by SimonBiggs** [comment]
Yup, early does suite. Potentially though we could clump together interruptions and do it one Thursday after the software development subgroup meeting?

**Comment by pchlap** [comment]
> Yup, early does suite. Potentially though we could clump together interruptions and do it one Thursday after the software development subgroup meeting?

Potentially, although we do a daily standup at 10am, so would need to jump off by then. Not sure how long this would take?

**Comment by SimonBiggs** [comment]
If everything goes exceptionally smoothly (unlikely) it might be able to fit in ~20 minutes. We could potentially just do it in two separate ~20 min blocks in that 9.30 -> 9.50 time slot?

**Comment by pchlap** [comment]
Yep, that would work. We could also do the 30 mins before the subgroup meeting. But let's wait until we're ready to proceed with the release and we'll lock something in :)

**Comment by SimonBiggs** [comment]
Yup, sounds good to me :)

**Comment by Matthew-Jennings** [comment]
Hi @pchlap & @SimonBiggs, I'll dial in if you'll have me (and you haven't met yet!)

**Comment by SimonBiggs** [comment]
That'd be awesome @Matthew-Jennings. Do you reckon while @pchlap is wading through the process and while I'm helping him you might be able to focus on writing up the release docs as we go?

**Comment by pchlap** [comment]
Hi @SimonBiggs & @Matthew-Jennings, this all sounds great. Since #1571 / #1575 has been merged, did we want to look at perhaps doing this on Thursday morning?

**Comment by SimonBiggs** [comment]
Ideally yes, but before embarking on it we need to have made sure that the
update tests pass:

https://github.com/pymedphys/pymedphys/runs/4278232900?check_suite_focus=true

And that all CVEs highlighted by dependabot have been fixed. The CVE
remaining from dependabot only effects our e2e testing suite, so isn't a
user facing issue at all. ...but... It's a nice habit to have all of those
cleared away before making a release...

I don't think I'll get the chance before this coming Thursday to make those
fixes. If either of you were in a position to have a gander at seeing
what's involved to fix those that'd be awesome :).

Otherwise let's give it one more week, that way I'll have a chance to jump
on during the weekend and fix it if no one else gets the chance.

**Comment by pchlap** [comment]
Sounds good @SimonBiggs, yes lets probably give it 1 more week then. I'll aim to jump and have a look to see if I can get my head around those tests but I doubt I'll be able to do that and solve the issue by Thursday morning.

**Comment by Matthew-Jennings** [comment]
> That'd be awesome @Matthew-Jennings. Do you reckon while @pchlap is wading through the process and while I'm helping him you might be able to focus on writing up the release docs as we go?

Great idea, @SimonBiggs, let's do it

**Comment by SimonBiggs** [comment]
I'll send through a calendar invite for 8am NSW time next Thursday :slightly_smiling_face:

**Comment by SimonBiggs** [comment]
I still haven't had a chance to fix those issues on the main branch, and I don't believe @cedricbelanger has yet had a chance to put his name on the front page.

@pchlap and @Matthew-Jennings would you both be okay to postpone one week?

**Comment by pchlap** [comment]
> I still haven't had a chance to fix those issues on the main branch, and I don't believe @cedricbelanger has yet had a chance to put his name on the front page.
> 
> @pchlap and @Matthew-Jennings would you both be okay to postpone one week?
No problem @SimonBiggs, that works well for me :)
